sdk/cosmos/azure-cosmos/azure/cosmos/_cosmos_client_connection.py

```diff
@@ -141,9 +143,12 @@ def __init__(
             # We need to set continuation as not expected.
             http_constants.HttpHeaders.IsContinuationExpected: False,
         }
-
+        self._user_agent = _utils.get_user_agent()
+        self.diagnostics = CosmosDiagnostics(ua=self._user_agent)
```

Comment:
Not sure why we still need this CosmosDiagnostics object?

Reply:
I am planning to remove cosmos diagnostics and any of the references I made for it.